examples/FluentUI.Demo.Shared/Pages/TextArea/Examples/TextAreaRowsAndCols.razor

```diff
@@ -0,0 +1,11 @@
+﻿﻿<p>
+    12 rows: <br />
+    <FluentTextArea @bind-Value=value1 Rows="12"></FluentTextArea>
```

Comment:
I think we need to use a consistent convention for using double quotes for attributes. Example: https://mathiasbynens.be/notes/unquoted-attribute-values

In future development, it might be better to use `attribute="value"`.